Fix path for "Send Emails To" config

makzef opened the pull request:
## Purpose of this pull request

This pull request (PR) fixes a typo in config path for "Send Emails To" setting.  
In devdocs was `contact/contact/recipient_email` path, which is not exists. The correct one is `contact/email/recipient_email`.

<img width="1096" alt="Знімок екрана 2022-01-31 о 16 43 51" src="https://user-images.githubusercontent.com/20201927/151814599-5bd1feeb-cd7a-4b29-a5a8-5e11d8fe01f0.png">

## Affected DevDocs pages

<!-- REQUIRED List the affected pages on devdocs.magento.com (URLs). Not needed for large numbers of files. -->

-  https://devdocs.magento.com/guides/v2.4/config-guide/prod/config-reference-most.html


<!--
If you are fixing a GitHub issue, note it using GitHub keyword format (https://help.github.com/en/articles/closing-issues-using-keywords#closing-an-issue-in-a-different-repository) to close the issue when this pull request is merged. Example: `Fixes #1234`

`master` is the default branch. Merged pull requests to `master` go live on the site automatically. Any requested changes to content on the `master` branch must be related to the released codebase. Any content related to future releases goes in the `develop` branch.

See Contribution guidelines (https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md) for more information.
-->
## Fixed Issues (if relevant)  

Fixes  magento/magento2#35003

BarnyShergold reviewed (approved):
This is a good fix but also needs to be applied to the 2.3 version of this file (I checked)
Please can you also update this file on this PR, then it can be approved

BarnyShergold reviewed (changes requested):
Approved in error

makzef commented:
> This is a good fix but also needs to be applied to the 2.3 version of this file (I checked) Please can you also update this file on this PR, then it can be approved

@BarnyShergold Hello) I added changes you mentioned and updated PR. Thank you!

dobooth commented:
Good eye, @makzef!

dobooth commented:
@magento import pr to magento-devdocs/devdocs

m2-github-services commented:
@dobooth the Pull Request is successfully imported.

BarnyShergold reviewed (approved):
Thank you!
